definition/data/sheets/CaseEvent.json

```diff
@@ -7,14 +7,14 @@
-  {"CaseTypeID": "MoneyClaimCase", "ID": "InterlocutoryJudgment", "Name": "Interlocutory Judgment Req", "Description": "Interlocutory Judgment Request", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;judgmentDecideAmount", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByDefendant", "Name": "Defendant referred judge", "Description": "Refer To Judge By Defendant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;redeterminationRequested", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByClaimant", "Name": "Claimant referred judge", "Description": "Refer To Judge By Claimant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;redeterminationRequested", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;judgmentRequested", "SecurityClassification": "Public",  "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "InterlocutoryJudgment", "Name": "Interlocutory Judgment Req", "Description": "Interlocutory Judgment Request", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "judgmentDecideAmount", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByDefendant", "Name": "Defendant referred judge", "Description": "Refer To Judge By Defendant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "redeterminationRequested", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByClaimant", "Name": "Claimant referred judge", "Description": "Refer To Judge By Claimant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "redeterminationRequested", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "judgmentRequested", "SecurityClassification": "Public"},
```

Comment:
ReferToJudgeByDefendant and  ReferToJudgeByClaimant are triggered after CCJ, so ideally the precondition for  those events should have judgmentRequested. Please test those journeys before committing